contracts/Core/interface/IVoteManager.sol

```diff
@@ -4,31 +4,85 @@ pragma solidity ^0.8.0;
 import "../../lib/Structs.sol";
 
 interface IVoteManager {
+    /**
+     * @notice stores the salt calculated in block manager
+     * @param _salt the hash of the last epoch and medians of the block
+     */
     function storeSalt(bytes32 _salt) external;
 
+    /**
+     * @notice stores the depth of a valid merkle tree. Depth of the merkle tree sent by the stakers should match with this
+     * for a valid commit/reveal
+     * @param _depth depth of the merkle tree
+     */
     function storeDepth(uint256 _depth) external;
 
+    /**
+     * @notice returns vote value of a collection reported by a particular staker
+     * @param epoch in which the staker reveal this value
+     * @param stakerId id of the staker
+     * @param medianIndex index of the collection
+     * @return vote value
+     */
     function getVoteValue(
         uint32 epoch,
         uint32 stakerId,
-        uint16 assetId
+        uint16 medianIndex
```

Comment:
please replace this with activeCollectionIndex
Its not medianIndex any more, as block is not given to have everything

Reply:
cant do that unless your PR is merged. will get linting error if i make this change right now